### ISSUE TITLE
Share course by email can't change reference to Richie

### ISSUE THREAD

**Issue by igobranco** [openfun/richie#1804]
## Bug Report

**Expected behavior/code**
A clear and concise description of what you expected to happen (or code).
The mailto subject and message should include the name of the platform (FUN-MOOC, NAU, etc.) instead of reference Richie.
The users don't understand what is Richie.

**Actual Behavior**
The mailto link includes `Richie`.

**Steps to Reproduce**
1. Open any course page
2. Click on mail button
3. The mailto has reference to Richie instead of the platform

**Environment**
- Richie version: any
- Platform: any

**Possible Solution**

Add new Django setting `PLATFORM_NAME` and use it when translating the mailto subject and body.

**Additional context/Screenshots**
![image](https://user-images.githubusercontent.com/67018/196796127-465bafc2-0769-4baa-9407-adbaae1b3479.png)

Our solution on NAU, was to force the translation of those strings and replace `Richie` to `NAU` on both en and pt-pt languages.

**Comment by sampaccoud** [comment]
If I'm not mistaken the idea was that it should be overriden via template override?

**Comment by igobranco** [comment]
Yes, it could be overridden using a template override. I saw that you have overide it on each of fun-richie-site-factory, but you override it on each to do exactly that, change the application name.
On NAU I've changed the translation of that string, that is another approach to do exactly the same thing.

I suggest to add new Django setting `PLATFORM_NAME`, like what Open edX has, and to use it when translating the mailto subject and body. With that we no longer need to override that template (Fun solution) or forcefully change the translations (NAU solution).

**Comment by igobranco** [comment]
Closing old issue.
